index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Image Resizer v.3</title>
+<style>p{text-align:center;}</style>
+</head>
+<body>
+       
+
+        
+        
+        
+    <div style="text-align: center;">
+    <div style="display: inline-block; text-align: left">
+<form action="/cgi-bin/image_resizer_web.py">
+        <br>
+  <h1>Image Resizer v.3.0 (alpha)</h1>
+  		<h2>Please, input the path of images<br>(or use the default):</h2>
+         Input Path: 
+         <input type="text" name="path_in" size="40" placeholder="/Users/garazd/Documents/Python/Img_In"><br>
+        Width: 
+        <input type="text" name="width" size="10" placeholder="600"><br>
```

Comment:
(adition to previous comment about default values)
Also we could put a default value using attribute value="your_value". But I'm not sure which variant is better (actually I forgot to ask about it at the lection).
Maybe Kostya will see this and answer.